go/cmd/vtctldclient/command/vreplication/movetables/movetables.go

```diff
@@ -49,6 +49,7 @@ func registerCommands(root *cobra.Command) {
 	create.Flags().BoolVar(&createOptions.AtomicCopy, "atomic-copy", false, "(EXPERIMENTAL) A single copy phase is run for all tables from the source. Use this, for example, if your source keyspace has tables which use foreign key constraints.")
 	create.Flags().StringVar(&createOptions.WorkflowOptions.TenantId, "tenant-id", "", "(EXPERIMENTAL) The tenant ID to use for the MoveTables workflow into a multi-tenant keyspace.")
 	create.Flags().BoolVar(&createOptions.WorkflowOptions.StripShardedAutoIncrement, "remove-sharded-auto-increment", true, "If moving the table(s) to a sharded keyspace, remove any auto_increment clauses when copying the schema to the target as sharded keyspaces should rely on either user/application generated values or Vitess sequences to ensure uniqueness.")
+	create.Flags().StringSliceVar(&createOptions.WorkflowOptions.Shards, "shards", nil, "Only create vreplication streams in these target shards. Ensure that that all rows on the source will route to the selected shards, as per your Vindex choice.")
```

Comment:
Typo: `Ensure that that all`

Reply:
I think we could word it more clearly so that the user understands that THEY should do this and that Vitess does not. Suggestion:
```
create.Flags().StringSliceVar(&createOptions.WorkflowOptions.Shards, "shards", nil, "Only create vreplication streams on the specified subset of target shards. Warning: you should first ensure that all rows on the source route to the specified subset of target shards using your VIndex of choice or you could lose data during the migration.")
```

This value is not used outside of multi-tenant workflows, right? If not, it's worth mentioning that as well IMO.